models/transaction.js

```diff
@@ -2,22 +2,45 @@ const Transaction = require('../schemas/transaction');
 const Customer = require('../schemas/customer');
 
 exports.createTransaction = async transaction => {
-  console.log(transaction);
-  let res = await Transaction.create({
+  const res = await Transaction.create({
     id: transaction.id,
     quantity: transaction.quantity,
     price: transaction.price,
     customerId: transaction.customerId,
     shipperId: transaction.shipperId,
-    coffeeId: transaction.coffeeId
+    coffeeId: transaction.coffeeId,
+    total: transaction.total,
+    status_code: transaction.status_code
   });
   return res;
 };
 
 exports.getCustomerAndTransactions = async id => {
-  let res = await Customer.findAll({
+  const res = await Customer.findAll({
     include: [{ model: Transaction }],
     where: { id: id }
   });
   return res;
 };
+
+exports.getAllTransactions = async () => {
+  const res = await Transaction.findAll();
+  return res;
+};
+
+exports.updateTransaction = async (info, id) => {
+  let updateValue = {};
```

Comment:
This is `const` too.

Reply:
Changing it means `updateValue=…`. If you modify a property of it you're mutating, not changing the value.